src/test/kotlin/org/rust/ide/inspections/RsCastToBoolInspectionTest.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+import org.rust.lang.utils.RsDiagnostic
+import org.rust.lang.utils.RsDiagnostic.CastAsBoolError.Companion.errorMessage
+
+class RsCastToBoolInspectionTest : RsInspectionsTestBase(RsCastToBoolInspection::class) {
+
+    fun testCastNumToBool() = checkByText("""
+        fn main() {
+            <error descr="$errorMessage [E0054]">5 as bool</error>;
+        }
+    """)
+
+    fun testCastBoolToBool() = checkByText("""
+        fn main() {
+            true as bool;
+        }
+    """)
```

Comment:
I wasn't sure how else to test this, but what is being tested here is that there *aren't* any errors in the provided code. I'd love to know if there's a better way to do it!

Reply:
It's exactly how we test absence of errors. If `RsCastToBoolInspection` start produce error in this case, test will fail